be/src/util/rle_encoding.h

```diff
@@ -856,6 +861,38 @@ inline bool RleBatchDecoder<T>::GetLiteralValues(int32_t num_literals_to_consume
     return true;
 }
 
+template <typename T>
+inline bool RleBatchDecoder<T>::SkipLiteralValues(int32_t num_literals_to_skip) {
+    int32_t num_consumed = 0;
+    if (HaveBufferedLiterals()) {
+        int32_t num_to_skip = std::min<int32_t>(num_literals_to_skip, num_buffered_literals_ - literal_buffer_pos_);
+        literal_buffer_pos_ += num_to_skip;
+        literal_count_ -= num_to_skip;
+        num_consumed = num_to_skip;
+    }
+
+    int32_t num_remaining = num_literals_to_skip - num_consumed;
+
+    int32_t num_to_bypass = std::min<int32_t>(literal_count_, BitUtil::RoundDownToPowerOf2(num_remaining, 32));
+    if (num_to_bypass) {
+        if (UNLIKELY(!bit_reader_.skip_bytes(num_to_bypass * bit_width_ / 8))) {
+            return false;
+        }
+        literal_count_ -= num_to_bypass;
+        num_consumed += num_to_bypass;
+        num_remaining = num_literals_to_skip - num_consumed;
+    }
+
+    // todo: lazy skip
```

Comment:
maybe we can perform lazily skip in the most upper level? such as column reader? so we only have to implement one lazily skip and do actual skip when we want to get a value